pennylane/devices/default_qubit.py

```diff
@@ -626,12 +627,11 @@ def _apply_state_vector(self, state, device_wires):
             raise ValueError("State vector must be of length 2**wires.")
 
         norm_error_message = "Sum of amplitudes-squared does not equal one."
-        if qml.math.get_interface(state) == "torch":
+        if not isinstance(
+            qml.math.linalg.norm(state, ord=2), jax.interpreters.partial_eval.DynamicJaxprTracer
+        ):
```

Comment:
I wonder if it is possible to avoid the `isinstance()` check, since this requires importing JAX 🤔